kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiStatement.java

```diff
@@ -543,6 +543,18 @@ public int getMaxRows() throws SQLException {
     return maxRows;
   }
 
+  @Override
+  public boolean getMoreResults(int current) throws SQLException {
+    // getMoreResults() javadoc says, that it should implicitly close
+    // any current ResultSet object, so here in case of Statement.CLOSE_CURRENT_RESULT
+    // we can implement the same logic
+    // see also https://issues.apache.org/jira/browse/HIVE-7680
```

Comment:
we usually use such words in comments for backport cases, but it's not. let's make it more clear.

please change the comment to something like
```suggestion
    // this is an additional enhancement on top of HIVE-7680
```